src/util.ts

```diff
@@ -84,6 +90,20 @@ export async function detectMinikube(pathAddition: string, installer: MinikubeIn
   return undefined;
 }
 
+// search if minikube is available in the path or in the extension folder
+export async function findMinikube(downloader: MinikubeDownload): Promise<string | undefined> {
+  try {
+    return await whereBinary('minikube');
+  } catch (err: unknown) {
+    console.debug(err);
+  }
+
+  const extensionPath = downloader.getMinikubeExtensionPath();
+  if (fs.existsSync(extensionPath)) {
+    return extensionPath;
+  }
+}
```

Comment:
I was wondering if the find should not be part of the object MinikubeDownload

b/c it seems people like to have tons of functions everywhere with tons of parameters but if we have something where we need to give the MinikubeDownload object maybe it can be attached to the MinikubeDownload object

Reply:
Yes, for sure, moving it to the MinikubeDownload!